boards/arm/nrf5340_dk_nrf5340/nrf5340_dk_nrf5340_cpunet.dts

```diff
@@ -118,25 +118,29 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		boot_partition: partition@1000000 {
+		/* partitions start offsets are expressed relative to
```

Comment:
@nvlsianpu @mbolivar  if this is an established convention in Zephyr, )i.e. the reg properties here denote relative addresses) we do not need the comment and shall remove it.

Reply:
I relay like this comment, but if you are thinking I should remove it, I will remove :D

Reply:
We're not having this in any other similar partitioning section (i've checked the ARM boards).
And I like to have consistency across ARM boards at least.

But I agree this is comment useful but should be somewhere where we explain the generic DTS-based partitioning framework (if we don't have such a section we should add it).

Reply:
> But I agree this is comment useful but should be somewhere where we explain the generic DTS-based partitioning framework (if we don't have such a section we should add it).

https://docs.zephyrproject.org/latest/guides/dts/index.html#defining-partitions

Reply:
thanks @mbolivar

Reply:
There is link above each partitions node to the mentioned doc. It is sufficient to understand partitions content. Although I personally appreciate to have hint-comment immediate in the code.